packages/@lwc/engine-core/src/framework/component.ts

```diff
@@ -15,7 +15,11 @@ import { Template, isUpdatingTemplate, getVMBeingRendered } from './template';
 import { VNodes } from './vnodes';
 import { checkVersionMismatch } from './check-version-mismatch';
 
-const signedTemplateMap: Map<LightningElementConstructor, Template> = new Map();
+type ComponentMetadata = {
+    tmpl: Template;
+    sel: string;
```

Comment:
Nitpick: I still don't get why we call this `sel` and not `tag`. But I know `sel` is all over the codebase, and this is internal-only, so it doesn't really matter.

Reply:
I'm not sure why we use `sel` either, I chose it to line up with what was already in the code base.  Originally I set it to `tagName` but I figured it would be better to match what we were already doing.